dashboard/views/_positions/_ec/_excuse.py

```diff
@@ -43,12 +43,13 @@ def excuse_quick_accept(request, position_slug, excuse_id):
 
 
 @verify_position([Position.PositionChoices.SECRETARY, Position.PositionChoices.VICE_PRESIDENT, Position.PositionChoices.PRESIDENT, Position.PositionChoices.ADVISER])
-def secretary_all_excuses(request):
-    """ Renders Excuse archive """
-    excuses = Excuse.objects.exclude(status='0').exclude(event__in=RecruitmentEvent.objects.all()).order_by('brother__last_name', 'event__date')
+def all_excuses(request, position_slug):
+    """ Renders Excuse archive for the given position """
```

Comment:
May want to specify here that the only valid positions right now are secretary and recruitment chair (could also do this on the `event_type_from_position` docstring.


I suppose at the very least the way this is written will make the issue very obvious when you pass in an invalid key though